### PR TITLE
Remove SetLogger() when new a DefaultLogger (Fixes#18)

### LINKED ISSUE #1: Umbrella issue for data-plane integrations with OpenSergo Go SDK
Umbrella issue for data-plane integrations with OpenSergo Go SDK.

In-progress:

- [ ] Sentinel Go
- [ ] CloudWeGo Kitex
- [ ] Kratos

### LINKED ISSUE #18: logging.NewXxxLogger will actually change the current logger, which seems confusing.
## Issue Description

Type: *enhancement*

### Describe what feature you want

`logging.NewXxxLogger` (e.g. `NewConsoleLogger`, `NewFileLogger`) will actually change the current logger, which seems confusing. By convention, a "New" function should create a logger without changing the current logger. We might need to separate the "set logger" from the "new logger" function.

`logging.NewXxxLogger` 命名与 "new" 含义不符，这里面其实包含两部分：create a logger, set the logger. 建议 NewLogger 仅包含创建逻辑，不要包含 SetLogger 操作。

### PR DESCRIPTION
**Pull request by jnan806**
Signed-off-by: Jiangnan Jia <jnan0806@gmail.com>

### Describe what this PR does / why we need it
Remove SetLogger() when new a DefaultLogger

### Does this pull request fix one issue?

Fixed #18 

### Special notes for reviews

### CODE REVIEW

File: pkg/common/logging/logger.go [hunk 1]
```diff
@@ -179,19 +174,23 @@ func ErrorWithCallerDepth(logger Logger, logFormat LogFormat, logCallerDepth int
...
+	loggers = append(loggers, loggerAppend)
+	return len(loggers)
 }
 
-// SetConsoleLogger set the consoleLogger to print int stdout
-func SetConsoleLogger(logger Logger) {
-	consoleLogger = logger
+// ClearLogger clear the Logger into loggerSlice
+func ClearLogger() {
+	loggers = make([]Logger, 0)
```
Comment (sczyh30) [inline pkg/common/logging/logger.go:193]: Should we wrap it with the mutex?

Comment (jnan806) [reply pkg/common/logging/logger.go:196]: sure.

**Comment by sczyh30** [comment]
Could you please resolve the conflicts?

**Comment by jnan806** [comment]
> Could you please resolve the conflicts?

done.

**Review by sczyh30 — APPROVED** [review]
LGTM

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**